README.md

```diff
@@ -47,6 +47,7 @@ For either platform, you can alternatively pass down a child element that will b
 | hideDoneBar             | boolean        | Hides the bar with tabbing arrows and Done link to exit the modal. While this is typical on `select` elements on the web, the [interface guidelines](https://developer.apple.com/ios/human-interface-guidelines/controls/pickers/) does not include it.                                                                           | N         | iOS           |
 | onUpArrow / onDownArrow | function       | _ Presence enables the corresponding arrow<br>_ Closes the picker<br>\* Calls the callback provided                                                                                                                                                                                                                               | N         | iOS           |
 | mode                    | string         | Specifies how to display the selection items when the user taps on the picker. 'dialog': Show a modal dialog. This is the default. 'dropdown': Shows a dropdown anchored to the picker view.                                                                                                                                      | N         | Android       |
+| onDonePress             | bool           | Callback when the 'done' button is pressed                                                                                                                                                                                                                                                                                        | N         | Both          |
```

Comment:
this should be a function and it's iOS only. bump it up a couple rows so it lives with the other iOS props.